ios/FluentUI.Demo/FluentUIDemoTests/ActivityIndicatorTest.swift

```diff
@@ -12,4 +12,27 @@ class ActivityIndicatorTest: BaseTest {
     func testLaunch() throws {
         XCTAssertTrue(app.navigationBars[controlName].exists)
     }
+
+    func testSizes() throws {
+        // ensures all 5 activity indicators sizes are shown
+        for i in 0...4 {
+            XCTAssert(app.images.containing(NSPredicate(format: "identifier MATCHES %@", "Activity Indicator.*size \(i).*")).element.exists)
+        }
+    }
+
+    // tests start/stop functionality as well as hiding (activity indicator should disappear when stopped)
+    func testStartStopHide() throws {
+        let startStopButton: XCUIElement = app.buttons["Start / Stop activity"]
+        let inProgress: NSPredicate = NSPredicate(format: "identifier CONTAINS %@", "Activity Indicator that is in progress")
+
+        XCTAssert(app.images.element(matching: inProgress).exists)
+        startStopButton.tap()
+        XCTAssert(!app.images.element(matching: inProgress).exists)
+
+        app.cells.containing(.staticText, identifier: "Hides when stopped").firstMatch.tap()
+        XCTAssert(app.images.element(matching: NSPredicate(format: "identifier CONTAINS %@", "Activity Indicator that is progress halted")).exists)
```

Comment:
Optional, I know the code here and in the line above is only used once, but putting startStopButton and inProgress into their own variables personally made it easier to read what the test is doing. Could make a "hidesWhenStoppedButton" and "halted" variable above

Reply:
Also seems like some of the other tests use the same NSPredicate, maybe you could define it as a class variable?